.storybook/stories/datalist/datalist.stories.ts

```diff
@@ -39,18 +27,37 @@ const defaultParameters: Parameters = {
     // story helpers
     elements,
     optionCount: elements.length,
-    content: 'Option',
+    placeholder: 'Options',
     disabled: false,
   },
 };
 
-const variants: Parameters[] = [
-  {
-    disabled: false,
-  },
-  {
-    disabled: true,
-  },
-];
+const DatalistTemplate: Story = args => ({
+  template: `
+    <clr-datalist-container>
+      <input clrDatalistInput [disabled]="disabled" [placeholder]="placeholder" />
+      <datalist>
+        <ng-container *ngFor="let element of elements; let i = index">
+          <option *ngIf="i < optionCount" [value]="element.symbol">{{element.name}}</option>
+        </ng-container>
+      </datalist>
+      <clr-control-helper>Helper text</clr-control-helper>
+      <clr-control-error>There was an error</clr-control-error>
+    </clr-datalist-container>
+  `,
+  props: args,
+});
+
+export const Datalist: StoryObj = {
+  render: DatalistTemplate,
+};
+
+export const Enabled: StoryObj = {
+  render: DatalistTemplate,
+  args: { disabled: false },
+};
```

Comment:
We don't need it as the default one have 'disabled: false' by default and we are not covering anything additional.
```suggestion
```